src/include/work_task.h

```diff
@@ -67,6 +67,11 @@ enum work_type {
 	                        */
 };
 
+enum wtask_delete_option {
+    DELETE_ONE,
+    DELETE_ALL
+};
```

Comment:
Well the enum is used only in a function which is meant to delete a work task... I can not think of a scenario where I would need a DELETE_NONE. But if you think it is for sanity purpose only then I can add it too.